diff --git a/node_modules/@strapi/upload/dist/admin/components/MediaLibraryInput/Carousel/CarouselAssets.js b/node_modules/@strapi/upload/dist/admin/components/MediaLibraryInput/Carousel/CarouselAssets.js
--- a/node_modules/@strapi/upload/dist/admin/components/MediaLibraryInput/Carousel/CarouselAssets.js
+++ b/node_modules/@strapi/upload/dist/admin/components/MediaLibraryInput/Carousel/CarouselAssets.js
@@ -86,7 +86,7 @@ const CarouselAssets = /*#__PURE__*/ React__namespace.forwardRef(({ assets, disa
                             setIsEditingAsset(true);
                             onDoubleClickAsset(asset, event);
                         },
-                        "aria-selected": currentAsset.id === asset.id,
+                        "aria-selected": currentAsset?.id === asset.id,
                         children: /*#__PURE__*/ jsxRuntime.jsx(CarouselAsset.CarouselAsset, {
                             asset: asset
                         })
